### ISSUE TITLE
Stuck "Processing..." pop-up with Bootstrap, caused by commit c0b73bdc434

### ISSUE THREAD

**Issue by domq** [aldeed/meteor-tabular#281]
1. Create sample Meteor app from the [example in the README.md](https://github.com/aldeed/meteor-tabular#example), fixing the obvious (i.e. `Books = new Mongo.Collection("Books");`)
2. `meteor add mrt:bootstrap-3`
3. Run app
4. Click on any column header

The "Processing..." pop-up shows up, and freezes (see screenshot)

Happens even on an empty table, and regardless of whether `tmpl` is in use in the columns.

`git bisect` gives c0b73bdc434 as the culprit.

![Screenshot](https://cloud.githubusercontent.com/assets/1629585/13546879/062d8e80-e2bc-11e5-9ec3-cdf2ccc5075a.png)

**Comment by domq** [comment]
Silly patch below "fixes" it, as well as #282

<pre>
diff --git a/client/tabular.js b/client/tabular.js
index 8894a9f..cd347fd 100644
--- a/client/tabular.js
+++ b/client/tabular.js
@@ -356,7 +356,7 @@ var tabularOnRendered = function () {
       }
     }
 
-    template.tabular.isLoading.set(false);
+    // template.tabular.isLoading.set(false);
   });
</pre>

**Comment by JuliaKoe** [comment]
So, how do you fix it?

**Comment by Manovsky** [comment]
I'm on the same problem. I'm actually using aslagle:reactive-table package and i would like to go for tabular package from aldeed... But i'm still locked on this processing popup.

here is my code : 

 ```
   TabularTables = {};

    TabularTables.printerJobs = new Tabular.Table({
        name: "printerJobs",
        collection: collPrinterJobs,
        pub: "printer.jobs",
        columns: [
            {data: "printerName", title: "Imprimante"},
            {data: "fileName", title: "Nom du fichier imprimé"},
            {data: "user", title: "Utilisateur"}
        ],
        language: {
            "zeroRecords": "no records !"
        }/*,
        processing: false*/      // when i put processing to false, i see my "no records !" message but not my data.
    });
```

and here is the html : 
`{{> tabular table=TabularTables.printerJobs class="table table-striped table-bordered table-condensed"}}`

Is there a problem in my code or need I doing a fix in tabular itself ?

**Comment by aldeed** [comment]
Should be fixed in 2.0
